### PR TITLE
Removed steady-state function and using ospsuite.parameteridentification

### LINKED ISSUE #26: getSteadyState from ospsuite.parameteridentification
getSteadyState() is currently duplicated in esqlabsR and ospsuite.parameteridentification. The main implementation sould be in ospsuite.parameteridentification, the one in esqlabsR removed.

### PR DESCRIPTION
**Pull request by PavelBal**
Fixes #26

### CODE REVIEW

File: R/esqlabs-env.R [hunk 1]
```diff
@@ -31,8 +35,8 @@ esqlabsEnv$heightPerPlotMapping <- 8
 #' getEsqlabsRSetting("packageVersion")
 #' getEsqlabsRSetting("packageName")
 getEsqlabsRSetting <- function(settingName) {
-  if (!(any(names(esqlabsEnv) == settingName))) {
-    stop(messages$errorEsqlabsRSettingNotFound(settingName))
+  if (!(any(names(ospsuiteEnv) == settingName))) {
+    stop(messages$errorPackageSettingNotFound(settingName, ospsuiteEnv))
```
Comment (IndrajeetPatil) [inline R/esqlabs-env.R:39]: Delete the message which is no longer used?

https://github.com/esqLABS/esqlabsR/blob/b17d6195eb7c48c77f3cebfaf6526ebe31f43291/R/messages.R#L16-L18

File: R/utilities-figures.R [hunk 2]
```diff
@@ -122,3 +122,27 @@ col2hsv <- function(color) {
...
+
+
+#' Create `DefaultPlotConfiguration` object with esqLABS defaults
+#'
+#' @return A `DefaultPlotConfiguration` object
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' plotConfig <- createEsqlabsPlotConfiguration()
```
Comment (IndrajeetPatil) [inline R/utilities-figures.R:134]: Run _styler_.

**Review by IndrajeetPatil — CHANGES_REQUESTED** [review]

File: R/esqlabs-env.R (hunk 1 above)
Comment (PavelBal) [reply R/esqlabs-env.R:39]: Yepp

File: man/esqLabsRSettingNames.Rd [hunk 3]
```diff
@@ -0,0 +1,16 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/esqlabs-env.R
+\docType{data}
+\name{esqLabsRSettingNames}
```
Comment (IndrajeetPatil) [inline man/esqLabsRSettingNames.Rd:4]: You need to redocument the functions. This should be `esqlabsRSettingNames`.

File: R/utilities-steady-state.R [hunk 4]
```diff
@@ -153,18 +17,17 @@ exportSteadyStateToXLS <- function(simulation,
                                    ignoreIfFormula = TRUE,
                                    stopIfNotFound = TRUE,
                                    lowerThreshold = 1e-15) {
-  validateIsOfType(simulation, type = "Simulation")
```
Comment (IndrajeetPatil) [inline R/utilities-steady-state.R:156]: This is removed because this validation happens in `getSteadyState()`?

**Review by IndrajeetPatil — CHANGES_REQUESTED** [review]

File: R/esqlabs-env.R [hunk 5]
```diff
@@ -31,8 +35,8 @@ esqlabsEnv$heightPerPlotMapping <- 8
 #' getEsqlabsRSetting("packageVersion")
```
Comment (IndrajeetPatil) [inline R/esqlabs-env.R:35]: This example doesn't work. 

```r
> getEsqlabsRSetting("packageVersion")
Error in getEsqlabsRSetting("packageVersion") : 
  object 'ospsuiteEnv' not found
```

Is this a copy-paste error? Shouldn't `ospsuiteEnv` be `esqlabsEnv` in the code below?

**Review by IndrajeetPatil — CHANGES_REQUESTED** [review]

**Review by IndrajeetPatil — APPROVED** [review]

**Pull request merged**